lib/rubocop/cop/style/frozen_string_literal_comment.rb

```diff
@@ -151,7 +151,7 @@ def last_special_comment(processed_source)
 
         def frozen_string_literal_comment(processed_source)
           processed_source.tokens.find do |token|
-            token.text.start_with?(FROZEN_STRING_LITERAL_REGEXP)
+            MagicComment.parse(token.text).frozen_string_literal_specified?
```

Comment:
`RuboCop::Cop::FrozenStringLiteral::FROZEN_STRING_LITERAL_REGEXP` constant can be removed. Can you remove it?

```diff
--- a/lib/rubocop/cop/mixin/frozen_string_literal.rb
+++ b/lib/rubocop/cop/mixin/frozen_string_literal.rb
@@ -6,7 +6,6 @@ module RuboCop
     module FrozenStringLiteral
       module_function

-      FROZEN_STRING_LITERAL_REGEXP = /#\s*frozen[-_]?string[-_]?literal:/i.freeze
       FROZEN_STRING_LITERAL_ENABLED = '# frozen_string_literal: true'
       FROZEN_STRING_LITERAL_TYPES_RUBY27 = %i[str dstr].freeze
```